pages/_app.tsx

```diff
@@ -3,61 +3,36 @@ import "@rainbow-me/rainbowkit/styles.css"
 import "react-toastify/dist/ReactToastify.css"
 
 import type { AppProps } from "next/app"
-import { RainbowKitProvider, darkTheme, getDefaultWallets } from "@rainbow-me/rainbowkit"
-import { configureChains, createConfig, WagmiConfig } from "wagmi"
-import { base, baseGoerli } from "@wagmi/core/chains"
-import { alchemyProvider } from "wagmi/providers/alchemy"
-import { publicProvider } from "wagmi/providers/public"
 import { ToastContainer } from "react-toastify"
 import { SessionProvider } from "next-auth/react"
 import * as React from "react"
 import { Analytics } from "@vercel/analytics/react"
+import { type PrivyClientConfig, PrivyProvider } from "@privy-io/react-auth"
 import { ThemeProvider } from "../providers/ThemeProvider"
-import { TITLE } from "../lib/consts"
 
-const isMainnet = !process.env.NEXT_PUBLIC_TESTNET
-const myChains = [isMainnet ? base : baseGoerli]
-const { chains, publicClient, webSocketPublicClient } = configureChains(myChains, [
-  alchemyProvider({ apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY }),
-  publicProvider(),
-])
-
-const { connectors } = getDefaultWallets({
-  appName: TITLE,
-  projectId: process.env.NEXT_PUBLIC_WALLETCONNECT_KEY,
-  chains,
-})
-
-const wagmiClient = createConfig({
-  autoConnect: true,
-  connectors,
-  publicClient,
-  webSocketPublicClient,
-})
+const privyConfig: PrivyClientConfig = {
+  loginMethods: ["email", "wallet"],
+  appearance: {
+    theme: "dark",
+    accentColor: "#FFFFFF",
+    logo: "/images/logo.jpg",
+  },
+  embeddedWallets: {
+    createOnLogin: "all-users",
```

Comment:
only create for users without an existing wallet
```suggestion
    createOnLogin: "users-without-wallets",
```
docs: https://docs.privy.io/guide/react/wallets/embedded/creation#automatic
<img width="747" alt="Screenshot 2024-03-22 at 11 03 09 AM" src="https://github.com/SweetmanTech/DACIRKUS_PERFORMERS/assets/23249402/99158ebc-cb5f-4b7b-be8e-d0d124309cdb">